intranet/static/js/eighth/user_link.js

```diff
@@ -56,11 +58,14 @@ $(function() {
         }, 350);
     });
 
-    $(document).mousemove(function(e){
-        var posx = window.mouseX = e.pageX;
-        var posy = window.mouseY = e.pageY;
+    $(document).mousemove(function(e) {
+        var posx = window.mouseX = e.pageX,
+            posy = window.mouseY = e.pageY;
         posx += 75;
         posy -= 215 - 10;
-        $("img.user-pic.active").css({left: posx, top: posy});
+        $("img.user-pic.active").css({
+            left: posx,
+            top: posy
+        });
     });
-})
+})
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://www.codacy.com/app/tjcsl/ion/file/2759572672/issues/source?bid=3176160&fileBranchId=3355598#l71)